bin/init.bash

```diff
@@ -156,6 +156,10 @@ set_storage_class() {
         aws)
             storage_class=ebs-gp2
             ;;
+
+        azure)
+            storage_class=azure-csi
+            ;;
```

Comment:
Could you set this up so that we have a `standard` storage class instead that we could use as default? For context: https://github.com/elastisys/ck8s-issue-tracker/issues/102

If you decide to keep it and given that we are considering adding both we would need to differentiate between `azuredisk` and `azurefile`.